src/views/DidUpgradeExplainer/DidUpgradeExplainer.tsx

```diff
@@ -74,12 +105,79 @@ export function DidUpgradeExplainer({ identity }: Props): JSX.Element | null {
             {t('view_DidUpgradeExplainer_deposit')}
           </p>
 
+          <section
+            className={
+              insufficientKilt
+                ? styles.paymentMethodsError
+                : styles.paymentMethods
+            }
+          >
+            <p className={styles.paymentMethod}>
+              <label
+                className={
+                  insufficientKilt ? styles.insufficientKilt : styles.kilt
+                }
+              >
+                <input
```

Comment:
Since we now have inputs here, would it make sense to make the outermost tag a `form`?